Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -460,7 +460,10 @@ def formula_version(formula, spec, contents = nil)
   end
 
   def check_open_pull_requests(formula, tap_full_name, args:)
-    GitHub.check_for_duplicate_pull_requests(formula.name, tap_full_name, state: "open", args: args)
+    GitHub.check_for_duplicate_pull_requests(formula.name, tap_full_name,
+                                             state: "open",
+                                             file:  formula.path.relative_path_from(formula.tap.path).to_s,
```

Comment:
Not all `formula` have non-nil `tap`s

Reply:
If I'm not mistaken, `brew bump-formula-pr` is not useful for formulae that don't have a tap?

Reply:
True, sorry, missed the method that this was being called on. As long as that case is handling in `brew bump-formula-pr` 👍🏻 from me.